api/api.js

```diff
@@ -0,0 +1,13 @@
+import axios from 'axios';
+
+class Api {
+    async getRequest(path) {
```

Comment:
I would say that getRequest can be static, no class is needed because class instance doesn't contain any particular details